ccm_web/client/src/pages/Home.tsx

```diff
@@ -57,7 +57,7 @@ function Home (props: HomeProps): JSX.Element {
     setCourseNameAsync,
     (course: CanvasCourseBase|undefined) => {
       props.setCourse(course)
-      enqueueSnackbar('Course name saved', { variant: 'success' })
+      enqueueSnackbar('Course name was saved! Please refresh the page to see the change in Canvas.', { variant: 'success' })
```

Comment:
Sorry, I'm still thinking about this. The word "see" could be problematic language (for visually impaired users). Is this clear enough?
```suggestion
      enqueueSnackbar('Course name was saved! Please refresh the page to have the change appear in Canvas.', { variant: 'success' })
```